angular-http-server.js

```diff
@@ -79,3 +78,23 @@ function start() {
         return console.log("Listening on " + getPort());
     });
 }
+
+function returnDistFile() {
+    var path = "index.html";
+    var argvPath = argv.path;
+    if (argvPath && fs.existsSync(argvPath)) {
+        path = argvPath + "/index.html";
```

Comment:
Use [`path.join(argvPath, 'index.html')`](https://nodejs.org/api/path.html#path_path_join_paths) so this is OS-agnostic

Reply:
👍